meddlr/data/transforms/transform.py

```diff
@@ -364,3 +365,211 @@ def __call__(self, kspace, maps, target, fname, slice_id, is_fixed, acceleration
         target = target.squeeze(0)
 
         return masked_kspace, maps, target, mean, std, norm
+
+
+class MotionDataTransform:
```

Comment:
We are only going to use this transform for inference right now - only keep the logic that is relevant when `is_test=True`